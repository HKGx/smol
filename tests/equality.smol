let a := 1
let b := 2

if a == b:
    print("a is equal to b")
else:
    print("a is not equal to b")


let c := 3
if b + 1 == c:
    print("b + 1 is equal to c")
else:
    print("b + 1 is not equal to c")