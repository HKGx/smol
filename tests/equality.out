a is not equal to b
b + 1 is equal to c
None
